dds_web/api/user.py

```diff
@@ -784,13 +784,75 @@ class SecondFactor(flask_restful.Resource):
     @handle_validation_errors
     def get(self):
 
-        token_schemas.TokenSchema().load(flask.request.json)
+        try:
+            token_schemas.TokenSchema().load(flask.request.json)
+        except marshmallow.ValidationError as err:
+            raise ddserr.AuthenticationError(message=err.messages)
```

Comment:
Isn't this already handled in the `@handle_validation_errors`?

Reply:
Done!